docs/deployments/deploy-chrome-extension.mdx

```diff
@@ -82,10 +76,10 @@ curl -X PATCH https://api.clerk.com/v1/instance \
   -d '{"allowed_origins": ["chrome-extension://<CHROME_EXTENSION_KEY>"]}'
 ```
 
-Running the command will add the extension's ID to your instance's `allowed_origins`.
-
 ## Submit your extension for review
 
-1. In your application, create a new zip of `build/chrome-mv3-dev` folder. Upload this to the Chrome Web Store Developer Dashboard.
-1. In the **Privacy** tab, find the **Permission justification** section. In this section, there is a **Are you using remote code?** section. You will need to indicate that you are using remote code. Clerk loads the `@clerk/clerk-js` package from a CDN and it is not included in the extension.
-1. Submit for review in the Chrome Developer Dashboard.
+1. In your app, create a new zip of the `build/chrome-mv3-dev` folder.
+1. In the top-right of the [Chrome Web Store Developer Dashboard](https://chrome.google.com/webstore/developer/dashboard), select **Upload new package** and upload the zip file.
+1. In the sidebar, select the **Privacy** tab.
+1. In the **Permission justification** section, it asks **Are you using remote code?**. You will need to indicate that you are using remote code. In the **Justification** textbox, paste "Clerk loads the `@clerk/clerk-js` package from a CDN and it is not included in the extension."
```

Comment:
1. In the **Permission justification** section, it asks **Are you using remote code?**. We have had limited reports that Google may reject the extension as there is code for other Clerk SDKs that loads the `@clerk/clerk-js` package from our CDN. The Chrome Extension SDK bundles the @clerk/clerk-js and does not load it from a remote source.

Reply:
Not sure what action is meant to be taken here

Reply:
Its a suggested edit for line 84. We don't hot load clerk-js with the Chrome Extension SDK -- its bundled.

Reply:
so in the **Are you using remote code?** box, you would like the user to write "Clerk has had limited reports that Google may reject the extension as there is code for other Clerk SDKs that loads the @clerk/clerk-js package from our CDN. The Chrome Extension SDK bundles the @clerk/clerk-js and does not load it from a remote source."

Reply:
I would like to advice them that Google might say something, and give them the information to pass on to Google.  I'm not really sure they need to do it right away -- checking that box makes it harder to get approved so better off not unless you get rejected for that reason and need to explain it.

Reply:
Looks like this was missed @alexisintech

Reply:
@royanger We should be able to remove this now.

Reply:
For sure. I will update.